server/src/test/java/io/crate/integrationtests/DDLIntegrationTest.java

```diff
@@ -861,13 +861,6 @@ public void testCreateTableWithCustomSchema() throws Exception {
         assertThat((String) response.rows()[0][0], is("a"));
     }
 
-    @Test
-    public void testCreateTableWithIllegalCustomSchemaCheckedByES() throws Exception {
-        assertThrowsMatches(() -> execute("create table \"AAA\".t (name string) with (number_of_replicas=0)"),
```

Comment:
Could we keep this but instead use _ or spaces as illegal variant?

Reply:
Thanks, added them back and now they test spaces.